src/GitUtils.ps1

```diff
@@ -415,6 +415,52 @@ function Get-SshAgent() {
     return 0
 }
 
+function Get-NativeSshAgent {
+    # The ssh.exe binary version must include "OpenSSH"
+    # The windows ssh-agent service must exist
+    $service = Get-Service ssh-agent
+    $executableMatches = Get-Command ssh.exe | ForEach-Object FileVersionInfo | Where-Object ProductVersion -match OpenSSH
+    $valid = $service -and $executableMatches
+
+    if ($valid) {
+        return $service;
+    }
+
+    return $false;
+}
+function Start-NativeSshAgent([switch]$Quiet) {
+    $service = Get-NativeSshAgent
+    
+    if (!$service) {
+        return $false;
+    }
+
+    # Enable the servivce if it's disabled and we're an admin
+    if ($service.StartupType -eq "Disabled") {
+        if (Test-Administrator) {
+            Set-Service $service -StartupType Automatic
+        }
+        else {
+            # Todo: Do we want to display a message, throw an exception or something else if the service is disabled?
+            write-host The ssh-agent service is disabled. Please start the service and try again.
```

Comment:
I think `Write-Error` is most appropriate, since the service failed to start? Even with `-Quiet`, a failure is a failure. I believe this also makes the following `return $true` unnecessary.

And general nitpick: casing should be `Write-Host`, and the message should be quoted.

Reply:
Fixed

Reply:
The `return $true` is still necessary even when using `Write-Error`